### ISSUE TITLE
Should there be a way to individually jump hands to a marker?

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/ratio-and-proportion#291]
Suggestion for https://github.com/phetsims/QA/issues/582.
I noticed that the ability to jump the hands to 0-9 is only available when both hands are being controlled and jumps both of them. I think it might be useful to jump one to one number and the other to a different number. For instance, the ratio is 1:2, you could jump the left hand to 2 and the right hand to 4. The easiest way (I think) to do this would be to have the 0-9 option available when focusing on individual hands, and having just that one jump in that case. If there is a reason that this isn't done, feel free to close.

**Comment by zepumph** [comment]
I'm almost positive that this will be too much added complexity the the design. Over to @emily-phet to confirm and feel free to close.

**Comment by terracoda** [comment]
@KatieWoe, thanks for the suggestion.
I have to agree with @zepumph that this shortcut is too complicated to implement. For example, a modifier key would be needed, and in addition a learner would still need to move their focus back and forth to execute the shortcut on each hand.

Moreover, at a design meeting a few weeks ago we discussed the different ways learners interact with the individual Left and Right Hands and the custom Both Hands interaction. We discussed if there were possible ways to easily align some of the shortcuts. We decided that this was not something we could tackle now, and decided on no further design changes.

Part of the challenge is that the sliders are native interactions. They already have built-in jumping commands. Learners can jump an individual hand by 2's by using Page Up and Pg Down keys, and they can jump a hand to 0 and 10 (min/max) with Home and End keys. They can take tiny steps by adding the Shift key (this is a custom PhET shortcut).

The Both Hands is a custom interaction, so we have more control over how creative we can get with it, but it also means we have to provide instructions some where. For this sim all shortcuts are communicated in the Keyboard Shortcuts dialog.

Considering @emily-phet is on vacation, I am comfortable not pursuing this and closing this issue.
